addons/web/static/src/js/widgets/name_and_signature.js

```diff
@@ -349,8 +349,7 @@ var NameAndSignature = Widget.extend({
      * @returns {string} cleaned name
      */
     _getCleanedName: function () {
-        var regexRemoveSpecialCharacters = /[^\w\u00E0-\u00FC-'" ]/g;
-        var text = this.getName().replace(regexRemoveSpecialCharacters, '');
```

Comment:
The regex actually allowed *some* accented characters (`\u00E0-\u00FC`). Removing the filtering entirely means we'll now allow the entire unicode set, no? I doubt the replacement fonts cover all of unicode, so certainly some amount of filtering is still needed.

Reply:
I have to admit that I don't see the use case of this filtering, I asked @tivisse for an explanation. I will adapt as soon as I understand what is the purpose. I agree that they probably not cover the all unicode but actually `Sébastien` and  `Éléonore` are not able to use an automatic signature and I don't understand why we should filter out the accented characters.

Reply:
Ok, I my cent just felt ! I will extend the range, I just missed the `^`  and I did not directly see that it allows this range.
During the manual test,  I thought that `Sébastien` was filtered but in fact the current fonts does not have the accented character `é`. Thanks for pointing my attention to that regex.

Reply:
> actually Sébastien and Éléonore are not able to use an automatic signature

Should work for Sébastien ("é" is U+00E9, possibly unless they're using Safari and it decides to use the NFD form, which is U+0065 U+0301), though it wouldn't work for Éléonore (É is U+00C9).

Reply:
An option could be to use unicode-aware regex, which were added in ES2015, possibly with the existing as a fallback if unicode-aware regex are not supported by the browser (because it's MSIE)?

Reply:
### How it started:
![Screenshot from 2020-11-20 10-53-21](https://user-images.githubusercontent.com/2039870/99786516-e2dbc900-2b1e-11eb-9483-8f0fa14639a5.png)

### How it ended 😁
![Screenshot from 2020-11-20 10-54-11](https://user-images.githubusercontent.com/2039870/99786574-f5560280-2b1e-11eb-8b22-60aa15bc1d28.png)

Reply:
Though there is the question as to what ranges the signature fonts support exactly e.g. allowing all letters would be possible for unicode-aware regex but would the font be able to auto-generate signatures for arabic or malayan names in their original script?

Reply:
> An option could be to use unicode-aware regex, which were added in ES2015, possibly with the existing as a fallback if unicode-aware regex are not supported by the browser (because it's MSIE)?

At that point, I'd just drop the filtering, it's not helping much. If the code point isn't supported by the chosen font you get an ugly fallback, and if you filter it out the signature is missing stuff, possibly everything. Neither is ideal, and it's inevitable.
Throwing more code at it won't solve it, so I say: less code ;-)

Reply:
> would the font be able to auto-generate signatures for arabic or malayan names in their original script?

You can see above that it can't even do cyrillic ("how it ended" is with this PR)

Reply:
I changed the regex in the meanwhile

Reply:
> > would the font be able to auto-generate signatures for arabic or malayan names in their original script?
> 
> You can see above that it can't even do cyrillic ("how it ended" is with this PR)

Ah right, so "Sébastien and Éléonore are not able to use an automatic signature" was because some of the fonts literally did not provide for non-ascii characters, so the sig' looked like nonsense, not that it'd strip unknown chars out.

Reply:
> Throwing more code at it won't solve it, so I say: less code ;-)

Well if it's the existing version of just filtering out arbitrary stuff then yeah. An alternative would be to filter out fonts which don't support all the characters in name, but I doubt that's easy to query so it'd be a *lot* more code, unless we send that info in the fonts response (though that's still a lot of code as we'd need to generate or extract valid ranges still).

Reply:
I prefer less code too. Unfiltered version of this PR (seems that some fonts may work):
![image](https://user-images.githubusercontent.com/334923/99804458-54297500-2b3b-11eb-8a89-26e75d20c529.png)

Filtered version :smile: :
![image](https://user-images.githubusercontent.com/334923/99804658-a074b500-2b3b-11eb-943c-91e74ed267fb.png)

I think that this filter prevents things that works to work and does not prevent ugly rendering to happen.
So, unless the filter has a special purpose, I would choose `no filter at all`.